ompi/attribute/attribute_predefined.c

```diff
@@ -165,24 +188,35 @@ int ompi_attr_create_predefined(void)
 
 int ompi_attr_free_predefined(void)
```

Comment:
The manipulation logic for the predefined attributes looks incorrect to me. Based on my understanding of the sessions they should have been protected by a refcount but apparently they are not. What did I missed ?

Reply:
i'll add in an atomic ref count.

Reply:
reviewing the code there actually already is a refcnt via the ompi_attr_get_ref/ompi_attr_put_ref  mechanism.